app/src/main/java/com/willowtree/vocable/presets/NumberPadFragment.kt

```diff
@@ -0,0 +1,74 @@
+package com.willowtree.vocable.presets
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.GridLayout
+import androidx.core.os.bundleOf
+import androidx.core.view.updateMargins
+import com.willowtree.vocable.BaseFragment
+import com.willowtree.vocable.R
+import com.willowtree.vocable.customviews.VocableButton
+import com.willowtree.vocable.databinding.FragmentNumberPadBinding
+import com.willowtree.vocable.databinding.PhraseButtonBinding
+import com.willowtree.vocable.room.Phrase
+
+class NumberPadFragment: BaseFragment() {
+
+    companion object {
+        private const val KEY_PHRASES = "KEY_PHRASES"
+        const val MAX_PHRASES = 12
+
+        fun newInstance(phrases: List<Phrase>): NumberPadFragment {
+            return NumberPadFragment().apply {
+                arguments = bundleOf(KEY_PHRASES to ArrayList(phrases))
+            }
+        }
+    }
+
+    private var binding: FragmentNumberPadBinding? = null
+    private var numColumns = 1
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        binding = FragmentNumberPadBinding.inflate(inflater, container, false)
```

Comment:
Do you see using this binding outside of `onCreateView`? 

If not, might be worth it to just make the binding a local variable of `onCreateView` - that way it's not nullable *and* you don't need to worry about nulling it out in `onDestroyView`